tests/test_uri_templates.py

```diff
@@ -602,3 +602,11 @@ def test_custom_error_on_suffix_route_not_found(client):
             resource_with_suffix_routes,
             suffix='bad-alt',
         )
+
+
+def test_custom_error_route_not_found(client):
+    class EmptyResource:
+        pass
+
+    with pytest.warns(UserWarning):
```

Comment:
If we leave the stonewall part, maybe this test becomes somewhat redundant. But, OTOH, it doesn't hurt.